packages/patternfly-4/content/design-guidelines/usage-and-behavior/embedded-assistance/design.md

```diff
@@ -0,0 +1,128 @@
+---
+path: "/design-guidelines/usage-and-behavior/embedded-assistance"
+---
+# Embedded assistance
+Use embedded assistance to provide additional help for users in context. Embedded assistance should be provided in a way that enables the user to find the information they need at the moment they need it while completing a particular step or task.
+
+## Principles
+**Use thoughtfully and sparingly**
+
+Never use embedded assistance as a substitute for usable and intuitive design.
+Embedded assistance should work to support and augment good user experience design.
+
+**Assume intermediate proficiency**
+
+Assume that the bulk of a users’ time in an application will be spent at a level of competency or proficiency rather than expert or novice.
+* Aim to defer novice information and reduce expert-level complexity.
+
+**Keep your content hierarchy in mind**
+
+Plan content hierarchy to provide the right content in the right place at the right time. You can think about content hierarchy in three levels:
```

Comment:
Would it help to link items here to more details about the specific elements in question? For example, if you reference pop-over link to where you discuss pop-overs. That will help connect these principles to the elements below in a holistic way.